pipeline_noted.ipynb

```diff
@@ -14,45 +14,56 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# About this document\n",
+    "<font size=\"6\">**About this document**</font>\n",
```

Comment:
why do you want to specify the formatting here instead of letting jupyter decide?